spec/features/audit/lists/filter_spec.rb

```diff
@@ -33,14 +29,17 @@
     end
 
     scenario "List filters by my unaudited content by default" do
-      visit audits_path
+      filter_audit_list = ContentAuditTool.new.filter_audit_list_page
+      filter_audit_list.load
 
-      expect(page).to have_select("allocated_to", selected: "Me")
-      expect(page).to have_checked_field("audit_status_non_audited")
+      filter_audit_list.filter_form do |form|
+        expect(form).to have_select("allocated_to", selected: "Me")
+        expect(form.audit_status).to have_checked_field("Not audited")
+      end
 
-      expect(page).to have_content("The Famous Five")
-      expect(page).to have_no_content("The Secret Seven")
-      expect(page).to have_no_content("The Wishing Chair")
+      expect(filter_audit_list).to have_list(text: "The Famous Five")
+      expect(filter_audit_list.list).to have_no_content("The Secret Seven")
```

Comment:
Can you do the following here:

```rb
expect(filter_audit_list).to have_no_list(text: "The Secret Seven")
```

(I think that is the documented approach – https://github.com/natritmeyer/site_prism#testing-that-an-element-does-not-exist)